app/namespaces/airtable_scraper/airtable_scraper_controller.py

```diff
@@ -49,7 +49,16 @@ def get(self):
             # Get all records from using airtable API
             records, status_code = get_all_records(visualize_on_serotracker_filter, airtable_fields_json_name)
             file_created_datetime = datetime.now()
-        result = {"airtable_request_status_code": status_code, "records": records, "updated_at": file_created_datetime}
+
+        max_sampling_end_date = datetime.min
+        for record in records:
+            if record["PUB_DATE"] is not None:
+                d = datetime.strptime(record["PUB_DATE"][0], '%Y-%m-%d')
+                if d > max_sampling_end_date:
+                    max_sampling_end_date = d
+        max_sampling_end_date = max_sampling_end_date.strftime('%Y-%m-%d')
+
+        result = {"airtable_request_status_code": status_code, "records": records, "updated_at": max_sampling_end_date}
```

Comment:
"updated_at" isn't really an accurate field name anymore, but leaving it in there for backwards compatibility

Reply:
alright, understandable